vllm/distributed/device_communicators/custom_all_reduce_utils.py

```diff
@@ -219,9 +218,13 @@ def gpu_p2p_access_check(src: int, tgt: int) -> bool:
         with tempfile.NamedTemporaryFile() as output_file:
             input_bytes = pickle.dumps(
                 (batch_src, batch_tgt, output_file.name))
+            # TODO(T212433680) make this sync'able
+            import libfb.py.parutil
             returned = subprocess.run([sys.executable, __file__],
```

Comment:
for the cases where `subprocess.run` is called elsewhere, I think you can export `PYTHONPATH=xxx` before starting vllm?